codec/doc.go

```diff
@@ -2,38 +2,26 @@
 // Use of this source code is governed by a MIT license found in the LICENSE file.
 
 /*
-Package codec provides a
```

Comment:
Most of these changes come from me copying the README.md into here to get them back in sync.